From: Andrea Righi <andrea.righi@canonical.com>
Subject: fix build on arches != amd64

Signed-off-by: Andrea Righi <andrea.righi@canonical.com>
---
 scheds/include/scx/common.bpf.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scheds/include/scx/common.bpf.h b/scheds/include/scx/common.bpf.h
index 5c503c2..f3410e2 100644
--- a/scheds/include/scx/common.bpf.h
+++ b/scheds/include/scx/common.bpf.h
@@ -10,7 +10,7 @@
 #include "vmlinux.h"
 #include <bpf/bpf_helpers.h>
 #include <bpf/bpf_tracing.h>
-#include <linux/errno.h>
+#include <asm-generic/errno.h>
 #include "user_exit_info.h"
 
 #define PF_WQ_WORKER			0x00000020	/* I'm a workqueue worker */
-- 
2.40.1

